src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/UnstructuredAddressTest.java

```diff
@@ -27,8 +27,9 @@ void testFromStringValidWithAllFields() {
         final var address = UnstructuredAddress.fromString("ADR++US:LINE1:LINE2:LINE3:LINE4:LINE5++POSTCODE");
         assertAll(
             () -> assertThat(address.getFormat()).isEqualTo("US"),
-            () -> assertThat(address.getAddressLines()).containsExactly("LINE1", "LINE2", "LINE3", "LINE4", "LINE5"),
-            () -> assertThat(address.getPostCode()).isEqualTo("POSTCODE")
+            () -> assertThat(address.getAddressLines())
```

Comment:
If we return a `String[]`/empty array here, then this assert can just be `.containsExactly("LINE1", "LINE2", "LINE3", "LINE4", "LINE5")`

Reply:
Sure! Fixed in other places withing this class as well. Thanks!
e77d7a364be138d0f6226ca0b07d00b792684bc2